<launch>
  <!-- Launch the parking controller in simulation. Launches controller node, and interactive marker node -->
  <node pkg="final_challenge" name="parking_controller" type="parking_controller.py" output="screen">
  	<param name="drive_topic" value="/drive" />
  </node>

  <node pkg="final_challenge" name="cone_sim_marker" type="cone_sim_marker.py" output="screen">
  </node>
</launch>